.vsts-ci-templates/push_perf_data.yml

```diff
@@ -17,3 +17,25 @@ steps:
       az storage blob upload-batch -d tx-times/$AGENT_NAME/${BUILD_SOURCEBRANCHNAME}_${BUILD_BUILDNUMBER} -s. --pattern "client_*_recv.csv.gz" --account-name ccfperf --subscription $(subscription)
       az storage blob upload-batch -d metrics/$AGENT_NAME/${BUILD_SOURCEBRANCHNAME}_${BUILD_BUILDNUMBER} -s. --pattern "*_metrics.json.gz" --account-name ccfperf --subscription $(subscription)
     workingDirectory: build
+
+# cimetrics related tasks: render and publish results for Pull Requests build
+- script: ./env.sh python -m cimetrics.plot
+  env:
+    METRICS_MONGO_CONNECTION: $(METRICS_MONGO_CONNECTION)
+  workingDirectory: build
+  displayName: 'Plot cimetrics'
+  condition: eq(variables['Build.Reason'], 'PullRequest')
+
+- script: ./env.sh python -m cimetrics.github_pr
```

Comment:
Very nice :)

Reply:
I wonder if we could distribute a yaml template for the steps inside the package, so it's even easier to set up?